### DISCUSSION TITLE
Allow water_heater entity in climate card

### DISCUSSION THREAD

**Discussion by faizpuru** [Clooos/Bubble-Card#955] [Feature requests]
Thanks for your project ! 

water_heater entities are not yet supported in the climate card.
These entities are really similar to climates: https://www.home-assistant.io/integrations/water_heater/

**Comment by Clooos** [comment]
Hi! Can you try to add it in YAML to see if it works like that? If yes I will just have to allow them in the entity list in the editor.

Edit: It seems that they are some differences, maybe not for the v2.3.0, I need to stop adding features in this beta 😅

**Reply by faizpuru** [reply]
The card display is working properly. However, the temperature change functionality  is currently not responding to user input. 
Let me know if you'd like me to provide more technical details.

**Reply by marcosg26** [reply]
Any update on this? I think the same issue of not being able to set the temperature is still there...

**Comment by Clooos** [comment]
Hi! This will be a new feature in v3.3.1, it was about time! 😄
